plugins/database/cassandra/cassandra.go

```diff
@@ -90,12 +91,13 @@ func (c *Cassandra) CreateUser(ctx context.Context, statements dbplugin.Statemen
 	}
 
 	creationCQL := statements.CreationStatements
```

Comment:
[Optional] It might read nice if `s/creationCQL/creationCQLStmts` and `s/rollbackCQL/rollbackCQLStmts` is made.